src/pages/index.js

```diff
@@ -7,9 +7,50 @@ import Section from "../components/Section.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupDelete from "../components/PopupDelete.js";
 import UserInfo from "../components/UserInfo.js";
 import "./index.css";
 import Constants from "../utils/constants.js";
+import Api from "../components/Api.js";
+
+// ! ||--------------------------------------------------------------------------------||
+// ! ||                                      API                                       ||
+// ! ||--------------------------------------------------------------------------------||
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7d9b26b0-78d0-422a-941a-09c95ea2b082",
+    "Content-Type": "application/json",
+  },
+});
+
+api.getUserInfo().then((userData) => {
+  profileInfo.setUserInfo({
+    name: userData.name,
+    description: userData.about,
+  });
+});
+
+api.getUserInfo().then((userData) => {
+  avatar.setAttribute("src", userData.avatar);
```

Comment:
Needs correction: please create a setter in the UserInfo class for the avatar `setAvatar` and use it here